### ISSUE TITLE
Running on ARM

### ISSUE THREAD

**Issue by warrenseine** [HubSpot/prettier-maven-plugin#66]
I'm running under Docker on macOS on an M1 chip. By default, Docker Desktop runs Linux on `linux/arm64/v8`, which is pretty cool.

However, the Node executable bundled with prettier-maven-plugin is built for x86_64 and fails with:
```
root@host:/var/code# /root/.m2/repository/com/hubspot/maven/plugins/prettier-maven-plugin/0.17/prettier-maven-plugin-0.17-node-12.13.0-linux.exe
qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory
```

When I explicit ask Docker to run with platform `linux/amd64`, it works, so there's a workaround. However, I'd be much better to have a compatible Node for this architecture.

I wonder if #64 would help here?

**Comment by warrenseine** [comment]
I see that indeed, this [seems to be the case](https://github.com/HubSpot/prettier-maven-plugin/pull/64/commits/548539ffd753820fa57639443af2362a8246995b#diff-a669e3b05be37663c85d38a042491eafdaacc1143b18ebfbe0c4a24f6493da44R43). Good luck with the merge!

**Comment by jhaber** [comment]
👍 should have that rolled out next week

**Comment by jhaber** [comment]
I just released this as part of version 0.18 by the way, in case you were waiting for a release

**Comment by warrenseine** [comment]
Coming back to this issue as it solved Darwin ARM64, but not Linux ARM64. This should be easily fixed with #104 now.
